ui/pages/settings/networks-tab/networks-form/networks-form.js

```diff
@@ -605,7 +605,7 @@ const NetworksForm = ({
               },
             },
             {
-              setActive: true,
+              setActive: false,
```

Comment:
This line could also just be removed since the default value for this argument is false.

Reply:
It seems no other caller of upsertNetworkConfiguration tries to override setActive. Let's just remove this and rely on the default